extensions/wikia/FeedsAndPosts/js/feedsAndPosts.js

```diff
@@ -0,0 +1,12 @@
+require(['jquery'], function ($) {
+	var fpLibrary = window.location.protocol + '//' + window.wgDBname + '.wikia.com/feeds-and-posts/public/dist/lib.min.js';
```

Comment:
what about wikis with the language code in the path?

Reply:
(Also, `wgDBname` does not represent a wiki's subdomain, so wgServer is probably better here)

Reply:
@mszabo-wikia It's EN only.